frontend/test/unit/specs/components/ImageDetails/v-copy-license.spec.js

```diff
@@ -20,16 +28,22 @@ describe("VCopyLicense", () => {
         license_url: "http://license.com",
         creator: "John",
         creator_url: "http://creator.com",
+        frontendMediaType: "image",
       },
       fullLicenseName: "LICENSE",
     }
     options = {
+      localVue,
       propsData: props,
+      i18n,
+      pinia: createPinia(),
     }
   })
 
-  it("should contain the correct contents", () => {
-    const wrapper = render(VCopyLicense, options)
-    expect(wrapper.find(".copy-license")).toBeDefined()
+  it("should contain the correct contents", async () => {
+    render(VCopyLicense, options)
+    expect(
+      screen.queryAllByText("media-details.reuse.copy-license.copy-text")
```

Comment:
Why do we not need an `await` here?

Reply:
`queryAllByText` is synchronous, as far as I can tell :thinking: I think only the "find" methods are async.